code/game/objects/structures/tables_racks.dm

```diff
@@ -42,7 +42,9 @@
 	. = ..()
 	if(_buildstack)
 		buildstack = _buildstack
-	AddElement(/datum/element/climbable)
+
+	if (!istype(src, /obj/structure/table/optable))
+		AddElement(/datum/element/climbable)
```

Comment:
Instead, have a variable for `climbable = TRUE` that op table sets to false.

Reply:
@Mothblocks Done, but I used "table_climbable" as the var name to prevent any confusion with the actual climbable datum. 

I also fixed a minor bug that I found while testing (pretty sure not caused by any of my PR's changes) where you could not unbuckle someone from an operating table while you were pulling them. This was caused by the way the code for the table placing mechanic works. I tested a few solutions but I found the easiest was just to remove the table placing mechanic on the operating table (or any table with table_climbable set to FALSE). You now have to either just buckle them or just shove them on.